### PR TITLE
Resizable: size and position properties can be modifed in resize event

### PR DESCRIPTION
**Pull request by scottgonzalez**
Fixes gh-220

### CODE REVIEW

File: entries/resizable.xml [hunk 1]
```diff
@@ -153,12 +153,24 @@
...
-					<desc>The current position represented as <code>{ left, top }</code></desc>
+					<desc>The current position represented as <code>{ left, top }</code>. The values may be changed to modify where the element will be positioned. Useful for custom resizing logic.</desc>
 				</property>
 				<property name="size" type="Object">
-					<desc>The current size represented as <code>{ width, height }</code></desc>
+					<desc>The current size represented as <code>{ width, height }</code>. The values may be changed to modify where the element will be positioned. Useful for custom resizing logic.</desc>
 				</property>
 			</argument>
+			<example>
+				<desc>Constrain horizontal resizing:</desc>
```
Comment (jzaefferer) [inline entries/resizable.xml]: This does the same thing as `maxWidth`, which isn't that great as an example. Here's a callback that I've been using in an application, restricting the resizable to a vertical grid of 30px:

```js
resize: function(event, ui) {
    ui.size.height -= ui.size.height % 30;
    ui.size.height += ui.originalPosition.top % 30;
}
```

Maybe add this in addition to the current example?

Comment (scottgonzalez) [inline entries/resizable.xml]: I have a feeling the complexity of what's actually happening there might trip up users who are reading through the docs. Perhaps something simpler, like just throttling size and assuming the top/left is already on the grid?

Comment (jzaefferer) [inline entries/resizable.xml]: Well, that's what you already have. Could add my example in addition to this one - if you think that is still too distracting, leave it as-is.

Comment (scottgonzalez) [inline entries/resizable.xml]: I meant something like `ui.size.height = Math.round( ui.size.height / 30 ) * 30;` to get the grid like snapping.

Comment (jzaefferer) [inline entries/resizable.xml]: Gotcha, that looks good.

**Comment by scottgonzalez** [comment]
Updated the example.

**Comment by jzaefferer** [comment]
:+1:

**Pull request closed without merging**